ncrystal_python/src/NCrystal/ncmat2endf.py

```diff
@@ -41,9 +41,13 @@
 from . import core as nc_core
 from . import constants as nc_constants
 from . import vdos as nc_vdos
+from ._common import print
 
 try:
-    import endf_parserpy
+    # TODO: temporary fix to avoid syntax warning from endf-parserpy
```

Comment:
Please add `https://github.com/IAEA-NDS/endf-parserpy/issues/10` in the comment as well, to make it clear later.